make.js

```diff
@@ -402,6 +239,195 @@ CLI.serverBuild = function() {
     banner('Build successful', true);
 }
 
+function buildTasks(taskName) {
```

Comment:
>buildTasks [](http://example.com/codeflow?start=9&length=10)

I think this should be "buildTask" instead of "buildsTasks"?  I only see the function building one task